rpcs3/util/atomic.cpp

```diff
@@ -933,7 +933,7 @@ template <typename F>
 FORCE_INLINE auto root_info::slot_search(uptr iptr, u128 mask, F func) noexcept
 {
 	u32 index = 0;
-	u32 total = 0;
+	[[maybe_unused]] u32 total = 0;
```

Comment:
why not remove it?

Reply:
It's a PR to fix warnings, not really to change code. It's probably going to be optimized out but I left it in case someone had it there to measure perf or such.